examples/test.cpp

```diff
@@ -1,6 +1,43 @@
-#include "../include/featurelib.h"
+#include "../include/feature_lib.h"
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
 
-int main() {
-  featurelib instance;
+std::vector<uint8_t> matToVector(const cv::Mat &mat) {
+  std::vector<uint8_t> result;
+  cv::Mat cont = mat.isContinuous() ? mat : mat.clone();  // make sure the Mat is continuous
+  cont.reshape(1, 1).copyTo(result);  // copy from a 1D Mat
+  return result;
+}
+//Please inter only filepath to command line arguments
```

Comment:
1. Add a space between `//` and the comment itself.
2. Fix typos: its "enter" not "inter".
3. You don't write docs like this: no need to write "please" -- you describe what the function does, what args it accepts, and what it returns. And you make it as short as possible. You can see some examples [in the code style](https://google.github.io/styleguide/cppguide.html#Function_Comments).

Reply:
done